src/java/org/apache/cassandra/db/ColumnFamilyStoreMBean.java

```diff
@@ -53,6 +54,7 @@
     /**
      * force a major compaction of specified key range in this column family
      */
+    @BreaksJMX("This API was released in 3.10 using a parameter that takes Range of Token, which can only be done IFF client has Cassandra binaries in the classpath")
```

Comment:
Shouldn't we provide a JMX-friendly variant of this method?

Reply:
sent out an email on the dev list to figure out what is best; waiting on that to touch these methods, so far only @ekaterinadimitrova2 has replied, and her feedback matches yours (fork)

Reply:
Yeah, I think we can mark as @Deprecated the old ones, without actually removing them, and add also new ones

Reply:
just pushed a `v2`.  We don't have the following `fn(range.toString()) == range`, so went with a "simpler" `token:token` string

Reply:
Looks good. I guess that the `v2` part of the name is to avoid overloaded methods. Since the method is applied to multiple token ranges, maybe we can call it `forceCompactionForTokenRanges`, in plural, so we don't keep the `v2` forever? Feel free to ignore if you don't agree.

Reply:
went with `s`